### DISCUSSION TITLE
Missing something when running on Ubuntu

### DISCUSSION THREAD

**Discussion by gwstern** [WebGoat/WebGoat#1492] [General]
I know I'm missing something obvious here. Created a Ubuntu VM, put Java and WebGoat on it but get:
> Exception in thread "main" java.lang.UnsupportedClassVersionError: org/owasp/webgoat/server/StartWebGoat has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
when I try to run it. Environment:
- openjdk version "11.0.18" 2023-01-17
- Ubuntu 22.04.2 LTS 
- WebGoat 2023.3 and 2023.4

**Comment by aolle** [comment]
You need Java 17. Please read the README.
